utils/dom.js

```diff
@@ -133,11 +129,29 @@ export function isVerticalEdge( container, isReverse, collapseRanges = false ) {
 	return true;
 }
 
+/**
+ * Get the rectangle of a given Range.
+ *
+ * @param {Range} range The range.
+ *
+ * @return {DOMRect} The rectangle.
+ */
+export function getClientRect( range ) {
```

Comment:
The name of the function and its placement in `utils/dom.js` doesn't seem to imply that I'd expect a value of type `Range` to be the argument of this function. Maybe we should include this somewhere in the name of the function?

Reply:
Sounds good!